spring-core/src/main/java/org/springframework/core/io/buffer/DataBufferUtils.java

```diff
@@ -572,8 +572,8 @@ public static Mono<DataBuffer> join(Publisher<? extends DataBuffer> dataBuffers)
 	public static Mono<DataBuffer> join(Publisher<? extends DataBuffer> buffers, int maxByteCount) {
 		Assert.notNull(buffers, "'dataBuffers' must not be null");
 
-		if (buffers instanceof Mono) {
-			return (Mono<DataBuffer>) buffers;
+		if (buffers instanceof Mono mono) {
```

Comment:
I want to add the generic wildcard to the `Mono`, but the latter return statement shows an error.
Should I add the generic wildcard to the `Mono`?

Reply:
>  Should I add the generic wildcard to the Mono?

I think we just have to leave that one as-is: a generic wildcard is not always possible.

In any case, I'll review it when I merge your work.